pkg/fiddler/fiddler.go

```diff
@@ -45,27 +45,30 @@ func GetCollections() []string {
 }
 
 // TransformRow transform a row of data into the coral schema
-func TransformRow(row map[string]interface{}, modelName string) ([]byte, error) {
+func TransformRow(row map[string]interface{}, modelName string) (interface{}, []byte, error) { // id row, transformation, error
```

Comment:
Consider not converting the returned row to a []byte and leave it as a map.  This way fiddler's role is just to transform the data and it's up to what ever comes next to decide how to use it.  (Also may make it unnecessary to return the id separately.)